src/elements/OcBreadcrumb.vue

```diff
@@ -92,15 +105,44 @@ export default {
       return [...this.items].reverse()[0]
     },
   },
+
+  created() {
+    // Add a general event listener to catch outside clicks
+    document.addEventListener("click", this.documentClick)
```

Comment:
I'm thinking about some nice way how to do this on a more global level. A custom directive comes to my mind but am not sure about an optimal way to bring that into the target projects (e.g. ocis-web)

Reply:
Register only when the dropdown is opened.